### PR TITLE
Fix JSQAudioMediaItem Outgoing design.

### PR DESCRIPTION
**Pull request by hassan8357**
## Pull request checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I have resolved any merge conflicts.
- [ ] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: ____

## What's in this pull request?
Fix JSQAudioMediaItem Outgoing design
Before
After

>...

### CODE REVIEW

File: JSQMessagesViewController/Model/JSQAudioMediaItem.m [hunk 1]
```diff
@@ -240,10 +240,18 @@ - (UIView *)mediaView
         playView.clipsToBounds = YES;
 
         // create the play button
-        CGRect buttonFrame = CGRectMake(self.audioViewAttributes.controlInsets.left,
-                                        self.audioViewAttributes.controlInsets.top,
-                                        self.audioViewAttributes.playButtonImage.size.width,
-                                        self.audioViewAttributes.playButtonImage.size.height);
+        CGRect buttonFrame;
+        if (!self.appliesMediaViewMaskAsOutgoing) {
+            buttonFrame = CGRectMake(self.audioViewAttributes.controlInsets.left + 5,
```
Comment (jessesquires) [inline JSQMessagesViewController/Model/JSQAudioMediaItem.m]: we don't want this hardcoded `5` here.

we should handle this somehow in `controlInsets`

**Comment by jessesquires** [comment]
Thanks so much @hassan8357 ! 😄 

@eliburke -- what's the best way to handle this?

**Comment by eliburke** [comment]
@hassan8357 thanks for catching this. 

@jessesquires I need to think on the best way to fix, but this isn't the first time I've bumped into alignment issues related to tail vs tailless bubbles. For example, aligning the cell top label with the bubble, rather than the bubble's tail, on an incoming message.

What do you think about adding a CGFloat for the tail width somewhere (perhaps JSQMessagesCollectionViewLayoutAttributes) and then I can start to look for places where we can use it to adjust for situations like this.

**Comment by jessesquires** [comment]
> What do you think about adding a CGFloat for the tail width somewhere

I don't think we'll need this for v8.0. Since we'll be section-based, the cell labels can become header/footer views.

**Comment by eliburke** [comment]
Ok, ignore the labels then, and just consider bubble contents. If i want a square image inside a bubble, or if I want to embed any view or html that does not get masked to the shape of the bubble (like here in the audio media bubble), the width of the tail is an unknown.

For now, would it be appropriate to use JSQMessagesCollectionViewLayoutAttributes.textViewFrameInsets.left?  (or textViewTextContainerInsets-- I can't remember which does what)

**Comment by jessesquires** [comment]
Oh derp. Sorry, I misread that.

**Comment by eliburke** [comment]
Replacement pull request in #1590.

**Comment by jessesquires** [comment]
thanks so much for finding and reporting this bug @hassan8357! 👍

**Pull request closed without merging**